.github/workflows/ci-test.yml

```diff
@@ -573,6 +904,8 @@ jobs:
           success="${{
             needs.prepare.result=='success'
             && (needs.dev-setup-sh-test.result=='success' || needs.dev-setup-sh-test.result=='skipped')
+            && (needs.build-images.result=='success' || needs.build-images.result=='skipped')
```

Comment:
I think we can skip this, since `land-block-test` needs `build-images`.

Reply:
nah, if build-images fails, land-block-test will be skipped.

Reply:
yea, I think I have discussed this point with Rex